wiki/Article_Style_Guide/en.md

```diff
@@ -1,192 +1,278 @@
+    - Linking to specific section of FAQ: `/wiki/FAQ#ive-never-played-this-ouendan-thing---what-is-it/`
+
+## Images
+**_Image display on the page:_**
+- Image display is as is without any formatting at all.
+  - It is best not to nest images inside GFM's table. (?)
+- For mobile display, hover-over text can be displayed by tapping on the image. The display of the hover-text may show up in extreme-left side of the mobile screen, rendering the alt-text unreadable. If the image was clickable, the hover-over text will not be displayed, and you will be directed to the link instead.
+  - Known to work (hover-over text): Google's Chrome
+  - The rest of the mobile browsers either does not support it, removed by compression (Opera Mini's Extreme Mode), or assumed you meant zoom-in/out (double-tap; depending on browser's settings).
+
+```Markdown
+Add an exclamation mark, '!', in front.
+![broken text](link "Alt text"), or   [broken text]: http://www.google.com "Omnipotent searchbar"
+                                      ...
+                                      ![broken text][my-reference]
+
+Alternatively, clickable image (from @MegaApplePi):
+[![alt-text](link-to-thumb.png) "Click to view full size"](link-to-full.png)
+
+or
+
+[thumb1]: link-to-thumb.png "Click to view full size"
+[full1]: link-to-full.png
+...
+[![alt-text][thumb1]][full1]
+```
+
+**_Image storage:_**
+- Put **images relevant only to the page** at the **same level as the page itself**.
+  - For older images, put in a deprecated folder (?).
+  - For images that will be used on other pages, put to `/wiki/shared` folder.
+  - Another way is to put the images in an `img` folder at the same page level (when there were too many images used).
+- To display the image on the page...
+  - An optimised thumbnail image should be displayed if the original image size is big (like screenshots of the game)
+  - **No formatting.**
+- Naming
+  - Prefix the image name with a page name it should be in.
+  - Name should _make sense_ to the image.
+  - Capitalisation and punctuation is strict (must be spelt exactly or _404 not found_) when directing to the image.
+    - Example 1 (same level): `./Interface_osu!catch.jpg`
+    - Example 2 (img folder): `./img/Interface_osu!catch.jpg`
+    - Example 3 (shared): `/wiki/shared/Ctb.gif`
+- Extension
+  - .png for original image size.
+  - .jpg for thumbnail image unless it has transparency.
+  - .gif for animation and icons.
+- For new image update, replace the old image using the same name and extension as the old image.
+  - Put old images to deprecated folder(?).
+- No idea about all the images from the MediaWiki by time of writing.
+  - May need to manually salvage the images from the old osu!wiki yourself.
+
+## Table styling
+Based from #70 :  
+Use Markdown table. Most of the Markdown will work except the unordered/ordered listing (must use HTML tags, see: <https://stackoverflow.com/questions/19950648/how-to-write-lists-inside-a-markdown-table>).
```

Comment:
Just to say, if you have to put a list inside a table... you are overdoing both the table and the list